### PR TITLE
Migrate from v2 to v3 of create-openapi-repo

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,7 @@ node_modules
 
 # npm
 package-lock.json
+
+
+# OpenAPI output bundles folder
+dist
```
Comment (ashkarpetin) [inline .gitignore]:
```suggestion
dist
```
```suggestion
dist

```

File: openapi/components/schemas/BankAccounts/BankAccountCreatePlain.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+type: object
+required:
+  - accountNumberType
+discriminator:
+  propertyName: accountNumberType
+  mapping:
+    BBAN: ./AccountNumberTypes/BBANType.yaml
+    IBAN: ./AccountNumberTypes/IBANType.yaml
+    BBANType: ./AccountNumberTypes/BBANType.yaml
+    IBANType: ./AccountNumberTypes/IBANType.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/BankAccounts/BankAccountCreatePlain.yaml]: BBANType/IBANType seems duplicates BBAN/IBAN

File: openapi/components/schemas/PaymentTokens/TokenBankAccountInstrument.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+type: object
+required:
+  - accountNumberType
+discriminator:
+  propertyName: accountNumberType
+  mapping:
+    BBAN: ./PaymentInstruments/BBANInstrument.yaml
+    IBAN: ./PaymentInstruments/IBANInstrument.yaml
+    BBANInstrument: ./PaymentInstruments/BBANInstrument.yaml
+    IBANInstrument: ./PaymentInstruments/IBANInstrument.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/TokenBankAccountInstrument.yaml]: `BBANInstrument/IBANInstrument` duplicates `BBAN/IBAN`

File: openapi/components/schemas/BankAccounts/BankAccountCreatePlain.yaml (hunk 2 above)
Comment (adamaltman) [reply openapi/components/schemas/BankAccounts/BankAccountCreatePlain.yaml]: Thanks for looking.

Comment (adamaltman) [reply openapi/components/schemas/BankAccounts/BankAccountCreatePlain.yaml]: I think you did find a flaw with the migration tool.  The discriminator mapped from two values to four values.  It should only have `BBAN` and `IBAN`.

Comment (RomanHotsiy) [reply openapi/components/schemas/BankAccounts/BankAccountCreatePlain.yaml]: Thanks. That's indeed a flaw with the migration tool. I will fix it asap.

**Comment by ashkarpetin** [comment]
Looks good, there is a conflict that needs to be resolved

**Pull request merged**